WordPress/Classes/ViewRelated/Support/SupportTableViewController.swift

```diff
@@ -1,10 +1,15 @@
-import UIKit
+import ContactUs
 import WordPressAuthenticator
 
 class SupportTableViewController: UITableViewController {
 
     // MARK: - Properties
 
+    lazy var contactSupportSelfServiceController = ContactSupportFlowController(
+        onSupportRequested: { [weak self] in self?.presentZendeskOrLoadForums() },
+        onHelpPageLoaded: { _ in }
+    )
```

Comment:
I decided to keep the contact us module agnostic of what support SDK we use and of how to log analytics events. It accepts closure to run to delegate those functionalities.